terraform/etl/55-aws-glue-icaseworks_ingest_etl.tf

```diff
@@ -0,0 +1,35 @@
+module "icaseworks_ingest_to_raw" {
+  source                    = "../modules/aws-glue-job"
+  is_production_environment = local.is_production_environment
+  is_live_environment       = local.is_live_environment
+
+  count = local.is_live_environment ? 0 : 0
```

Comment:
this resolves to `0` for all environments. 

for deploying to stg should be `count = !local.is_production_environment && local.is_live_environment ? 1 : 0`
for production and stg: `count = local.is_live_environment ? 1 : 0`